packages/teleterm/src/ui/QuickInput/useQuickInput.ts

```diff
@@ -16,16 +16,20 @@ limitations under the License.
 
 import React, { useEffect } from 'react';
 import { useAppContext } from 'teleterm/ui/appContextProvider';
-import { useKeyboardShortcuts } from 'teleterm/ui/services/keyboardShortcuts';
+import { useKeyboardShortcuts, useKeyboardShortcutFormatters } from 'teleterm/ui/services/keyboardShortcuts';
 import {
   AutocompleteResult,
   AutocompletePartialMatch,
 } from 'teleterm/ui/services/quickInput/types';
 import { routing } from 'teleterm/ui/uri';
+import { KeyboardShortcutType } from 'teleterm/services/config';
 
 export default function useQuickInput() {
-  const { quickInputService, workspacesService, commandLauncher } =
-    useAppContext();
+  const {
+    quickInputService,
+    workspacesService,
+    commandLauncher,
+  } = useAppContext();
```

Comment:
Could you check if you have Prettier configured properly? 😏

In the PTY PR I noticed that my editor formatted some of your changes, so when I saw this I decided to double check.

`yarn prettier -c packages/teleterm/src/ui/QuickInput/useQuickInput.ts` emits warnings, if I save the buffer in my editor the only remaining warning is about `jsxBracketSameLine` being deprecated.